### ISSUE TITLE
Enable branch protection

### ISSUE THREAD

**Issue by dajmcdon** [EpiForeSITE/summrt#4]
* Suggest we protect `main` to require a PR with 1 review and passing checks. 
* Possibly @gvegayon as CODEOWNER?

**Comment by gvegayon** [comment]
Good idea, @dajmcdon; I would suggest you, @annecori, and @cmilando as well, at least for the R bits, as you know better what's going on with the actual packages.
